3ds Max/Max2Babylon/Forms/AnimationGroupControl.cs

```diff
@@ -157,8 +157,11 @@ private void confirmButton_Click(object sender, EventArgs e)
             confirmedInfo.FrameStart = newFrameStart;
             confirmedInfo.FrameEnd = newFrameEnd;
 
-            if(nodesChanged)
+            if (nodesChanged)
+            {
                 confirmedInfo.NodeHandles = newHandles;
+                confirmedInfo = CalculateEndFrameFromAnimationGroupNodes(confirmedInfo);
```

Comment:
Here, we override the UI values with whatever value we calculate. I'm not sure if that's good UX.

Can we change this to prepopulate the text field with our "best guess of the end frame" on animation group creation, rather than render the text field useless?

Reply:
We should never assume the intent of the user. When I am building an animation timeline, there are definite areas in the timeline I never intend on exporting and if my values in the animation group UI are overwritten with calculated values, I will see that as a bug.

We should help the user by populating with our best guess to save them time, but we should never change what the user set in the UI. If the user enters values that are incorrect (I have keys to frame 50 and export to frame 100) they should see half of the clip with no transforms changing rather than assuming they didn't want those extra frame and eliminating them. There may be a valid reason for them to export their animation in this way that we cannot foresee.

Reply:
I will move this calculation on a specific a button, at least the user will be able to add many bones and at the end, the time range can be updated manually. is it ok for you?

Reply:
That's fine with me, as long as the user can always tweak whatever we calculate to their needs, that's my largest concern.